panel/tests/command/test_serve.py

```diff
@@ -0,0 +1,108 @@
+import contextlib
+import os
+import re
+import subprocess
+import sys
+
+import pytest
+import requests
+
+from queue import Empty, Queue
+from threading import Thread
+
+
+@contextlib.contextmanager
+def run_panel_serve(args):
+    cmd = [sys.executable, "-m", "panel", "serve"] + args
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=False)
+    try:
+        yield p
+    except Exception as e:
+        p.terminate()
+        p.wait()
+        print("An error occurred: %s", e)
+        try:
+            out = p.stdout.read().decode()
+            print("\n---- subprocess stdout follows ----\n")
+            print(out)
```

Comment:
Are these print statements captured by pytest and displayed when running the tests?

Reply:
They are if the test fails.